pvlib/iotools/pvgis.py

```diff
@@ -23,6 +23,347 @@
+    including satellite-based (SARAH, CMSAF, and NSRDB PSM3) and re-analysis
+    products (ERA5 and COSMO). Each data source has a different geographical
+    coverage and time stamp convention, e.g., SARAH and CMSAF provide
+    instantaneous values, whereas values from ERA5 are averages for the hour.
+
+    Notes
+    -----
+    data includes the following fields:
+
+    ===========================  ======  ======================================
+    raw, mapped                  Format  Description
+    ===========================  ======  ======================================
+    *Mapped field names are returned when the map_variables argument is True*
+    ---------------------------------------------------------------------------
+    P†                           float   PV system power (W)
+    G(i), poa_global‡            float   Global irradiance on inclined plane (W/m^2)
+    Gb(i), poa_direct‡           float   Beam (direct) irradiance on inclined plane (W/m^2)
+    Gd(i), poa_sky_diffuse‡      float   Diffuse irradiance on inclined plane (W/m^2)
+    Gr(i), poa_ground_diffuse‡   float   Reflected irradiance on inclined plane (W/m^2)
+    H_sun, solar_elevation       float   Sun height/elevation (degrees)
+    T2m, temp_air                float   Air temperature at 2 m (degrees Celsius)
+    WS10m, wind_speed            float   Wind speed at 10 m (m/s)
+    Int                          int     Solar radiation reconstructed (1/0)
+    ===========================  ======  ======================================
+
+    †P (PV system power) is only returned when pvcalculation=True.
+
+    ‡Gb(i), Gd(i), and Gr(i) are returned when components=True, otherwise the
+    sum of the three components, G(i), is returned.
+
+    See Also
+    --------
+    pvlib.iotools.read_pvgis_hourly, pvlib.iotools.get_pvgis_tmy
+
+    References
+    ----------
+    .. [1] `PVGIS <https://ec.europa.eu/jrc/en/pvgis>`_
+    .. [2] `PVGIS Hourly Radiation
+       <https://ec.europa.eu/jrc/en/PVGIS/tools/hourly-radiation>`_
+    .. [3] `PVGIS Non-interactive service
+       <https://ec.europa.eu/jrc/en/PVGIS/docs/noninteractive>`_
+    .. [4] `PVGIS horizon profile tool
+       <https://ec.europa.eu/jrc/en/PVGIS/tools/horizon>`_
+    """  # noqa: E501
+    # use requests to format the query string by passing params dictionary
+    params = {'lat': latitude, 'lon': longitude, 'outputformat': outputformat,
+              'angle': surface_tilt, 'aspect': surface_azimuth,
+              'pvcalculation': int(pvcalculation),
+              'pvtechchoice': pvtechchoice, 'mountingplace': mountingplace,
+              'trackingtype': trackingtype, 'components': int(components),
+              'usehorizon': int(usehorizon),
+              'optimalangles': int(optimalangles),
+              'optimalinclination': int(optimalangles), 'loss': loss}
+    # pvgis only takes 0 for False, and 1 for True, not strings, also the
```

Comment:
maybe incomplete comment

Reply:
It was left from a previously deleted comment.